src/ergw_aaa_radius.erl

```diff
@@ -37,12 +37,12 @@
 %% RFC: https://tools.ietf.org/html/rfc2866#section-5.10
 -define(DEFAULT_TERMINATION_CAUSE_MAPPING, [
     {normal, 1},
-    {administrative, 6},
-    {link_broken, 2},
-    {upf_failure, 2},
-    {session_error, 2},
+    {administrative, 10},
+    {link_broken, 9},
+    {upf_failure, 9},
+    {session_error, 9},
     {inactivity_timeout, 4},
-    {path_restart, 2}
+    {path_restart, 9}
 ]).
```

Comment:
I'm not convinced that this correct, not matter what a customer might want, the default should be reasonably allowed with RFC 2866.

e.g. administrative should really be 6, link_broken could be 2 or 3.

Reply:
Done